### ISSUE TITLE
Generate .md in the same vue components folder

### ISSUE THREAD

**Issue by equinusocio** [vuese/vuese#17]
Hi, is there a way to generate `.md` files beside the `vue` SFC components? I have this structure and i want to generate each md in the same component folder:

```sh
src/components
├── Button
│   ├── Button.story.js
│   └── Button.vue
├── Icon
│   ├── Icon.story.js
│   └── Icon.vue
├── Skeleton
│   ├── Skeleton.story.js
└── └── Skeleton.vue
```

**Comment by HcySunYang** [comment]
In fact, I have realized the benefits of this scenario and it is planned for the future.

**Comment by HcySunYang** [comment]
Yes, this is a good suggestion, but unfortunately, it is not supported now.

**Comment by HcySunYang** [comment]
You can install `@vuese/cli@^2.0.0-beta.8` which supports this feature.

1、Remove the previous global installation:
```js
yarn global remove vuese 
```
2、Install `@vuese/cli`:
```js
yarn global add @vuese/cli@^2.0.0-beta.8
```
3、Run the following command:
```sh
vuese gen --genType="markdown" --markdownDir="*"
```
The first available version of 2.0 will be released soon, you can view the 2.x roadmap for more planning: https://github.com/vuese/roadmap

 thanks
